java/src/main/java/com/powsybl/python/network/NetworkModificationsCFunctions.java

```diff
@@ -327,4 +327,37 @@ private static List<ConnectablePosition.Feeder> get3wtFeeders(ConnectablePositio
         }
         return feeders;
     }
+
+    @CEntryPoint(name = "replaceTeePointByVoltageLevelOnLine")
+    public static void replaceTeePointByVoltageLevelOnLine(IsolateThread thread, ObjectHandle networkHandle,
+                                                           CCharPointer teePointLine1,
+                                                           CCharPointer teePointLine2,
+                                                           CCharPointer teePointLineToRemove,
+                                                           CCharPointer bbsOrBusId,
+                                                           CCharPointer newLine1Id,
+                                                           CCharPointer newLine1Name,
+                                                           CCharPointer newLine2Id,
+                                                           CCharPointer newLine2Name,
+                                                           PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+
+        String teePointLine1Str = CTypeUtil.toString(teePointLine1);
+        String teePointLineStr = CTypeUtil.toString(teePointLine2);
+        String teePointLineToRemoveStr = CTypeUtil.toStringOrNull(teePointLineToRemove);
```

Comment:
@obrix @sylvlecl the builder has requireNonNull on several arguments do we let the builder make all the checks or we use CTypeUtil.toString() ?

Reply:
No strong opinion here, making the builder do the real check will probably give more explicit and easy to spot errors.

Reply:
@geofjamg @sylvlecl ?

Reply:
It's often better to try catch errors as early as possible, but it's OK for me this way:
the code of the builder is already in charge of checking arguments validity, let's not duplicate this role.

For ex. the builder could provide a meaningful error message, in that case it's better to let this through instead of having a plain null check before (but in the current code, it does not provide any such message, though :) ).

Reply:
ok